app-emulation/genymotion-bin/genymotion-bin-3.7.1.ebuild

```diff
@@ -127,20 +128,24 @@ src_install() {
 
 	if has_version app-emulation/qemu ; then
 		dodir /opt/"${MY_PN}"/qemu/bin
```

Comment:
Dumb question perhaps -- and totally unrelated to this version bump.

But, why can't the symlink be created regardless? Does genymotion treat a broken symlink as equivalent to none at all, or does it break in some special way?

It feels a bit awkward to have to re-emerge the package after installing QEMU. :( Especially given that binpackage caching would be broken by this.

Reply:
> But, why can't the symlink be created regardless? Does genymotion treat a broken symlink as equivalent to none at all, or does it break in some special way?

I simply didn't think broken links were allowed.

> It feels a bit awkward to have to re-emerge the package after installing QEMU. :( Especially given that binpackage caching would be broken by this.

Yes I agree.
Later today I will fix the ebuild.

Reply:
My feeling was that broken links *will* result in a QA warning, because they are usually a sign of a missing dependency, but that overall this is better than having the package result in different contents depending on which of the `|| ( ... )` dependencies are installed.

In this case, the QA warning would be a harbinger of an optional dependency.

Reply:
The updated ewarn mention of "re-emerge genymotion-bin to create the correct symbolic links" can be retracted, now, I think.